dev-docs/bidders/silverpush.md

```diff
@@ -124,6 +123,44 @@ var videoAdUnits = [{
 }]
 ```
 
-## Additional Details
+### First Party Data
+Publishers should use the `ortb2` method of setting First Party Data. The following fields are supported:
+
+- ortb2.user.ext.eids
+
+## Prebid Server
+
+```javascript
+pbjs.setConfig({
+    s2sConfig : {
+        accountId:"someaccountId" // replace with accountId
+        bidders : ['silverpush'],
+        timeout : 1000, //default value is 1000
+        adapter : 'prebidServer', //if we have any other s2s adapter, default value is s2s
+        endpoint: 'https://prebid.adnxs.com/pbs/v1/openrtb2/auction',
```

Comment:
Please make the endpoint generic here -- e.g. "https://PREBID_SERVER_ENDPOINT"

Reply:
Fixed it
update Commit :
[bb20ddb](https://github.com/prebid/prebid.github.io/pull/4748/commits/bb20ddbab18a0c81c108f805b2e7614e77cabfbf)